server/services/pageService.ts

```diff
@@ -1,16 +0,0 @@
-import { Response } from 'express'
```

Comment:
@davidmiddletonmoj @danielburnley @brightonsbox, are we all happy to move away from the page service? In hindsight is there any value in it?

Reply:
I think it makes sense for now - If we were calling the prisoner search service in the main route for the page itself we'd be doing a redundant call here. I could see moving it to the routes to end up needing a refactoring as we always call the header data, but in my opinion we should just let evolve naturally as our service does rather than try figure it out now given we're not sure what will/won't be needed by other routes yet